content/manual/running/hosting.md

```diff
@@ -7,196 +7,491 @@ weight = 2
+
+#### Create Droplet
+
+Hit this button to create the droplet.
+
+## 2. Get domain
+
+To access your ship easily from any device, it's necessary to have a domain
+name. You can either buy one from a domain registrar like
+[gandi.net](https://www.gandi.net/), [Namecheap](https://www.namecheap.com),
+etc, or you can get a free subdomain from a site like
+[freedns.afraid.org](https://freedns.afraid.org/).
+
+In this guide, we'll walk through the second free option, but if you'd prefer
+your own, you just need to add an A Record pointing to your droplet's public IP
+address.
+
+Go to [freedns.afraid.org](https://freedns.afraid.org/) and sign up. Once done,
+log in and select the "Subdomains" option in the menu on the left. Choose to
+add a new one, and you'll be presented with a screen like so:
+
+![afraid.org subdomain add](https://media.urbit.org/operators/manual/running/hosting/afraid-domain-add.png)
+
+You can put anything in the "Subdomain" field, but typically you'd put your
+planet name. You can choose whichever domain option you'd like. In the
+"Destination" field, you'll need to put the public IP address of your droplet,
+which you can get from the dashboard on Digital Ocean.
+
+Once you hit "Save", the configuration is complete:
+
+![afraid.org subdomain created](https://media.urbit.org/operators/manual/running/hosting/afraid-domain-add.png)
+
+## 3. Prepare for upload
+
+{% callout %}
+
+**Note**
+
+This step is necessary if you already have a ship running locally and want to
+move it to the cloud. If you don't, you can skip this step.
+
+{% /callout %}
+
+{% tabs %}
+
+{% tab label="If you're running from the command line" %}
+
+In the Dojo, use either `"CTRL + D"` or `|exit` to shut down your ship.
+
+Archive your pier by running `tar cvzf riclen-tinlyr.tar.gz ~/path/to/your/pier`
+(substitute your own ship name and pier location).
+
+{% /tab %}
+
+{% tab label="If you're using Port" %}
+
+Shut down your local ship and export it:
+
+In port, first click `"Home"` in the bottom left corner of the home page.
+
+![landscape
+screenshot](https://media.urbit.org/operators/manual/running/hosting/landscape-screenshot.png)
+
+Then, click `"Manage"` next to the planet you want to upload to the cloud.
+![port manage
+screenshot](https://media.urbit.org/operators/manual/running/hosting/port-manage-screenshot.jpg)
+
+Finally, click `"Eject"`, and make note of the location in which you export
+your archived planet so that you can use it in the next step.
+
+![port eject screenshot](https://media.urbit.org/operators/manual/running/hosting/port-eject-screenshot.jpg)
+  
+{% /tab %}
+
+{% /tabs %}
+
+## 4. Connect to the server
+
+To make connecting simple, you can add an alias to `~/.ssh/config` on your local
+machine. Open `~/.ssh/config` in an editor (you may need to create it if the
+file doesn't exist), and add the following to the bottom of the file (replacing
+the ship name and IP address with your own):
+
+``` {% copy=true %}
```

Comment:
partial indentation seems odd

Reply:
[this is the correct formatting for the ssh config file](https://linuxize.com/post/using-the-ssh-config-file/#ssh-config-file-structure-and-patterns), the indented lines are options for the host directive

Reply:
Ah, I didn't know this part:  "Indentation is not required but is recommended since it makes the file easier to read."